### DISCUSSION TITLE
How to build a PRM/exe binary package

### DISCUSSION THREAD

**Discussion by peeyee** [dbeaver/dbeaver#21502] [Q&A] [answered]
Hi, I meet two questions:
1. How to build a exe/rpm package of dbeaver? 
2. I've runed the command `mvn clean package` and got a zip file. How to install it on windows, or ubuntu system?

**Comment by flambchap** [comment, marked as answer]
`mvn clean package` produces portable versions of DBeaver in `product/community/target/products/org.jkiss.dbeaver.core.product`. You can move these artifacts anywhere on your disc and use them. Unfortunately, maven cannot produce exe, rpm, or zip files.
